### PR TITLE
Add question to update to paid package for angular

### LINKED ISSUE #724: Add a question to Step by Step mode to change package
### Description  
Add a question to Step by Step mode to change package on first project run. Applies to both main cli and schematics workflow, depends on #723

### PR DESCRIPTION
**Pull request by wnvko**
Now in wizard we will ask if user will use paid package. If so update the `package.json`.

Closes #724 

Additional information related to this pull request:

### CODE REVIEW

File: packages/core/prompt/BasePromptSession.ts [hunk 1]
```diff
@@ -369,6 +372,22 @@ export abstract class BasePromptSession {
 			break;
 		case "Complete & Run":
 			const config = ProjectConfig.localConfig();
+
+			if (config.project.framework === "angular" && !config.packagesInstalled) {
+				// TODO: should we add check if there are paid components at all
+				const usePaidAngular = await this.getUserInput({
+					type: "list",
+					name: "usePaidAngular",
+					message: "Do you have paid subscription:",
```
Comment (ViktorSlavov) [inline packages/core/prompt/BasePromptSession.ts:381]: Nit pick, but the name variable should be `useLicensedAngular` IMO.
Also, minor typo
```suggestion
					message: "Do you have a paid subscription:",
```

Comment (wnvko) [reply packages/core/prompt/BasePromptSession.ts:381]: we should revise very carefully all the messages in this PR. The existing messages are just a place holders :)

File: packages/ng-schematics/src/prompt/SchematicsPromptSession.ts [hunk 2]
```diff
@@ -13,6 +13,15 @@ export class SchematicsPromptSession extends BasePromptSession {
...
 	public projectName: string;
 
+	public get userAnswers(): Map<string, any> {
+		return this._userAnswers;
+	}
+	public set userAnswers(v: Map<string, any>) {
+		this._userAnswers = v;
+	}
+
+	private _userAnswers: Map<string, any>;
```
Comment (ViktorSlavov) [inline packages/ng-schematics/src/prompt/SchematicsPromptSession.ts:23]: Does this really need to be a private property w/ accessors?

Comment (damyanpetev) [reply packages/ng-schematics/src/prompt/SchematicsPromptSession.ts:23]: I asked as well 😁 @wnvko Mentioned something about the test spy on property messing up without accessors. Will try without it just to check

Comment (wnvko) [reply packages/ng-schematics/src/prompt/SchematicsPromptSession.ts]: done

File: packages/ng-schematics/src/ng-new/index.ts [hunk 3]
```diff
@@ -149,10 +154,21 @@ export function newProject(options: OptionsSchema): Rule {
 					}
```
Comment (ViktorSlavov) [inline packages/ng-schematics/src/ng-new/index.ts:154]: If the below call to the `upgradeIgniteUIPackages` method is set here, I believe you won't need to call `setVirtual` again

**Pull request merged**